core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -85,6 +85,7 @@ public class UtilControllerService implements InitializingBean {
   @Value("${klaw.coral.enabled:false}")
   private boolean coralEnabled;
 
+  private boolean isUserSuperadmin;
```

Comment:
in this case we shouldn't make it a class variable we can assign it on line 309 as 
boolean isUserSuperadmin = authenticationInfo.getUserrole().equals("SUPERADMIN");

Reply:
the rest looks good!